.github/workflows/PR_merged.yaml

```diff
@@ -0,0 +1,48 @@
+## Workflow for retrieve Release Notes section from PRs
+
+name: CI
+
+# Controls when the workflow will run
+on:
+   pull_request:
+    branches: [master]
+    types: [closed]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      # Getting PR information from the commit SHA
+      - name: Getting PR information
+        run: |
+          echo "Installing requirements"
+          sudo apt update
+          sudo apt install jq -y
+          sudo apt install curl -y
+          bash
+          echo "PR number is ${{ github.event.pull_request.number }}"
+          echo "Getting PR URL"
+          curl "https://api.github.com/repos/DynamoDS/Dynamo/pulls/${{ github.event.pull_request.number }}" | jq '._links.html.href' | sed -e 's/"/ /g' | sed 's/ //g' > pr_url_convert.out
+          cat pr_url_convert.out | sed -e 's/\//\\\//g' > pr_url.out
+          prurl=$(cat pr_url.out)
+          echo "Getting Release Notes Section from PR body"
+          curl "https://api.github.com/repos/DynamoDS/Dynamo/pulls/${{ github.event.pull_request.number }}" | jq '.body' | sed -e 's/\\r\\n/ /g' | sed -n -e 's/^.*### Release Notes //p' | sed -n -e 's/### Reviewers.*$//p' > releasenotes.out
+          releasenotes=$(cat releasenotes.out)
+          echo "Building JSON message"
+          cat << EOF > ./slack_message.json
```

Comment:
where is this file?

Reply:
The JSON file is created in the ubuntu VM provisioned.

Reply:
Sorry @carlosgzze I'm not understanding, is this .json file created during the run of the github action, or is it created manually beforehand somehow? If beforehand, I think it should be part of the repo or committed somewhere.

Reply:
Did you modify the secret to contain the full URL? Your last instructions to me where to only store the endpoint, so it does not include the domain.

Reply:
The JSON file is created during the run of github action and is built using the secrets, PR url and Release Notes section retrieved from PR body.
And it's a good point, I have updated the command to use only the last part of webhook url. Thanks for the heads up.

Reply:
The JSON file is created during the run of the github action and is built using secrets, PR URL and Release Notes section.